content/plotly/concepts/graph-objects/terms/surface/surface.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: '.Surface()'
+Description: 'Creates a 3d plot using the Surface() method of the graph_object_module in Plotly'
+Subjects:
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Plotly'
+  - 'Graphs'
+  - 'Data'
+  - 'Values'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.Surface()` function creates a 3D surface plot using Plotly's `graph_objects` module. It allows for the visualization of data in three dimensions.
+
+
+## Syntax
+
+```python
+plotly.graph_objects.Surface(
+    z=None,
+    x=None,
+    y=None,
+    colorscale=None,
+    cmin=None,
+    cmax=None,
+    opacity=None,
+    surfacecolor=None,
+    **kwargs
+)
+```

```

Comment:
```suggestion
```pseudo
plotly.graph_objects.Surface(
    z=None,
    x=None,
    y=None,
    colorscale=None,
    cmin=None,
    cmax=None,
    opacity=None,
    surfacecolor=None,
    **kwargs
)
```

```
The syntax in docs is written inside the ```pseudo ``` blocks, not the ```python``` blocks. You can have a look at the [style-guide ](documentation/style-guide.md) for better information and understanding.